testing/btest/scripts/base/protocols/rdp/rdpeudp2-handshake-success.zeek

```diff
@@ -0,0 +1,25 @@
+# @TEST-EXEC: zeek -r $TRACES/rdp/rdpeudp2-handshake-success.pcap %INPUT >out
+# @TEST-EXEC: btest-diff conn.log
+# @TEST-EXEC: btest-diff out
```

Comment:
* The baseline for this wasn't included in the PR, can generate via `btest -U scripts/base/protocols/rdp/rdpeudp2-handshake-success.zeek` and add/commit the new files in `Baseline/`.  (Also CI shows the previous test for `rdpeudp-handshake-success.zeek` should update its baseline)
* I noticed this one's output shows only the `rdpeudp_syn` event, but nothing else, were you expecting that or should it really be triggering the synac/established/data events, too?

Reply:
I've updated the btest Baselines. The events are now being raised by rdpeudp2-handshake-success.zeek.